pkg/iscsi/iscsi_util.go

```diff
@@ -459,9 +472,10 @@ func (util *ISCSIUtil) DetachDisk(
 	c iscsiDiskUnmounter,
 	targetPath string,
 ) error {
+
 	_, cnt, err := mount.GetDeviceNameFromMount(c.mounter, targetPath)
```

Comment:
Now, we can remove this line since we are checking in the below lines.

Reply:
yes, removed